### PR TITLE
SONARIAC-686 Create Entry point for all our custom assertions

### PR DESCRIPTION
**Pull request by mstachniuk**

### CODE REVIEW

File: iac-extensions/docker/src/test/java/org/sonar/iac/docker/tree/impl/AddInstructionImplTest.java [hunk 1]
```diff
@@ -63,107 +63,107 @@ void addInstructionSimple() {
     AddInstruction tree = parse("ADD src dest", DockerLexicalGrammar.ADD);
     assertThat(tree.getKind()).isEqualTo(DockerTree.Kind.ADD);
     assertThat(tree.keyword().value()).isEqualTo("ADD");
-    assertTextRange(tree.textRange()).hasRange(1, 0, 1, 12);
+    IacCommonAssertions.assertThat(tree.textRange()).hasRange(1, 0, 1, 12);
```
Comment (nils-werner-sonarsource) [inline iac-extensions/docker/src/test/java/org/sonar/iac/docker/tree/impl/AddInstructionImplTest.java:66]: Why is there no static import here compared to the last changes?

File: iac-extensions/docker/src/test/java/org/sonar/iac/docker/tree/impl/AddInstructionImplTest.java [hunk 2]
```diff
@@ -63,107 +63,107 @@ void addInstructionSimple() {
...
-    ArgumentAssert.assertThat(tree.dest()).hasValue("dest");
+    assertThat(tree.dest()).hasValue("dest");
   }
 
   @Test
   void addInstructionMultipleSrc() {
     AddInstruction tree = parse("ADD src1 src2 dest", DockerLexicalGrammar.ADD);
     assertThat(tree.getKind()).isEqualTo(DockerTree.Kind.ADD);
-    assertTextRange(tree.textRange()).hasRange(1, 0, 1, 18);
+    IacCommonAssertions.assertThat(tree.textRange()).hasRange(1, 0, 1, 18);
```
Comment (nils-werner-sonarsource) [inline iac-extensions/docker/src/test/java/org/sonar/iac/docker/tree/impl/AddInstructionImplTest.java:137]: Static import?

**Review by nils-werner-sonarsource — APPROVED** [review]
LGTM! Nice improvement. Please check my comment. You are not consistent in using static imports for assertThat. Is there a reason?

**Comment by mstachniuk** [comment]
It was an oversight. Fixed.

**Pull request merged**